src/app/components/dashboard/dashboard.component.scss

```diff
@@ -75,5 +75,5 @@
     .cb-files-container {
       columns: 1;
     }
-  } 
```

Comment:
Sure!
Ya, I will keep this in mind for the future as well.

Reply:
If this is the case, you can just add the required file and ignore the unnecessary changes using `git add <file_name1> <file_name2>`. This will make only the necessary files that will be mentioned in this command to go in the staging and finally the commit :)

Reply:
Great tip!. Thanks, I Will surely keep that in mind.